spring-cloud-gcp-samples/spring-cloud-gcp-pubsub-bus-config-sample/spring-cloud-gcp-pubsub-bus-config-sample-client/src/main/resources/application.properties

```diff
@@ -3,3 +3,4 @@
 
 #spring.cloud.gcp.project-id=[YOUR_GCP_PROJECT_ID]
 #spring.cloud.gcp.credentials.location=file:[LOCAL_PATH_TO_CREDENTIALS]
+spring.config.import=configserver:
```

Comment:
Ah qq what is the purpose of this line?

Reply:
This property indicates we need to import an external config from (local) config server.

You can read [Spring Cloud Config](https://docs.spring.io/spring-cloud-config/docs/current/reference/html/#_spring_cloud_config_client) for more info. This is similar with [what I did in Secret Manager](https://github.com/GoogleCloudPlatform/spring-cloud-gcp/pull/1204).